ansible/roles/ocp-workload-quarkus-cafe-demo-role/tasks/amq-streams-install.yml

```diff
@@ -0,0 +1,55 @@
+- name: Copying over AMQ Subscription
+  template:
+    src: templates/amq-sub.yaml.j2
+    dest:  "{{ config_location }}/amq-sub.yaml"
+    owner: "{{ default_owner }}"
+    group: "{{ default_group }}"
+    mode: 0666
+
+- name: Create Subscription to AMQ
+  command: "oc create -f {{ config_location }}/amq-sub.yaml"
+  register: rbac_result
+
+- name: Waiting for  AMQ Subscription to complete
+  shell: |
```

Comment:
Please avoid using the shell module. You should be able to get this with `k8s_info`.